universum_log_collapser/e2e/live_log_html.test.js

```diff
@@ -0,0 +1,28 @@
+const utils = require("./utils");
+
+beforeAll(async () => {
+    await utils.openPage("generated.html");
+    const expFailedStepsCount = 2; // depends on Universum test configuration
+    await utils.waitForHandledResults(expFailedStepsCount);
+});
+
+test('failed steps class changed', async () => {
+    await utils.checkFailedStepsHandled();
+});
+
+test("failed steps labels colored", async() => {
```

Comment:
Not a big deal, but the formatting of () is inconsistent: sometimes you add a space before and sometimes you don't.